src/BugYield/HarvestAdapter/HarvestAdapterApi.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace BugYield\HarvestAdapter;
+
+use Harvest\HarvestApi;
+use Harvest\Model\DayEntry;
+
+class HarvestAdapterApi extends HarvestApi {
+
+  /**
+   * Gets the entry specified
+   *
+   * Exists in adapter because: we want to have "multiple users"
+   * support (?of_user=).
+   *
+   * <code>
+   * $entry_id = 12345;
+   *
+   * $api = new HarvestApi();
+   *
+   * $result = $api->getEntry($entry_id);
+   * if ($result->isSuccess()) {
+   *     $entry = $result->data;
+   * }
+   * </code>
```

Comment:
It's a direct copy from the original DocBlock and modified with the new $user_id parameter. It's basically just implemented to make it as close to the original as possible, so you didn't have to look up any further documentation. But it could easily be removed :)